server/build.gradle

```diff
@@ -39,10 +39,10 @@ plugins {
 
 java {
     toolchain {
-        languageVersion.set(JavaLanguageVersion.of('21'))
+        languageVersion.set(JavaLanguageVersion.of(javaTargetVersion.toString()))
```

Comment:
@ajt001 Can I check that you have tested that the Gradle build picks up these variables? 

When you switched to using the variable previously, the build failed for John and me, which is why we explicitly set it to 21. If you've tested the build and it works for you, then leave it as it is; otherwise, please use 17 explicitly.

Reply:
@sadatmalik I always run the tests and builds from the command line (I prefer that than IDE), and had no issues. 
In any case, I've committed in changes to specify directly with JavaVersion.VERSION_17.

I have also added the 4 other build files so they're all consistent.